src/features/presence/PresenceAvatarList.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../styles/variables';
```

Comment:
Would could import this within the shared `./styles` file to avoid importing it in both components' stylesheets.